patches/server/0266-Hook-into-CB-plugin-rewrites.patch

```diff
@@ -167,18 +167,15 @@ index a679133a3de51e26eb19932ece9ade292879aefd..89a32de0a3fbb7465b078b2aa1cc1d15
                          if ( owner.equals( "org/bukkit/block/Biome" ) )
                          {
                              switch ( name )
-@@ -306,6 +420,14 @@ public class Commodore
-                             return;
+@@ -307,6 +421,11 @@ public class Commodore
                          }
  
-+                        // Paper start - Rewrite plugins
+                         // Paper start - Rewrite plugins
 +                        owner = getOriginalOrRewrite( owner) ;
```

Comment:
Is this spacing intentional?

Reply:
After looking again at it, it seems to be intentional

Reply:
partially intentional due to the existing code formatting in that class; somebody needs to pick a format though in their patch ;P